### ISSUE TITLE
niceSelect is not a function

### ISSUE THREAD

**Issue by arsensokolov** [hernansartorio/jquery-nice-select#60]
After use webpack and vue.js show this error, please resolve this problem. More about this problem on [Stackoverflow](https://stackoverflow.com/questions/46524680/jquery-nice-select-with-vue-js-is-not-a-function).

**Comment by ckizer** [comment]
Is there an update on this? I have the same issue with no solution.

**Comment by arsensokolov** [comment]
yes, I resolve this problem, I just rewrite code this plugin and pull request GH-62

you just use my version.

**Comment by M165437** [comment]
This plugin depends on jQuery ^2.2.3. If you require a newer version of jQuery in your project, try an alias to prevent webpack from resolving the old jQuery for this plugin.

```js
// webpack.config.js
module.exports = {
  //...
  resolve: {
    alias: {
      jquery: path.resolve('node_modules/jquery/dist/jquery.js')
    }
  }
};
```
